charts/cray-etcd-base/templates/certificates.yaml

```diff
@@ -23,21 +23,23 @@ OTHER DEALINGS IN THE SOFTWARE.
 */}}
 {{- if .Values.etcd.auth.client.secureTransport -}}
 ---
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
   name: "{{ include "cray-etcd-base.fullname" . }}-etcd-server-tls"
 spec:
   secretName: "{{ include "cray-etcd-base.fullname" . }}-etcd-server-tls"
   duration: 720h
   renewBefore: 24h
-  organization:
+  subject:
+    organizations:
     - Cray
```

Comment:
You should indent this 2 spaces to be consistent with the rest of the file

Reply:
fixed